src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -138,6 +138,8 @@ export default class ProcessConverter extends AbstractConverter<Process> {
   }
 }
 
+type Event = ShapeBpmnElementKind.EVENT_START | ShapeBpmnElementKind.EVENT_END;
```

Comment:
I suggest moving type definition on top of the file

Reply:
done, see b14af2b